pkg/dao/types/property/field.go

```diff
@@ -488,7 +488,7 @@ func (i Properties) Cty() (cty.Type, cty.Value, error) {
 	for x := range i {
 		t, v, err := i[x].Cty()
 		if err != nil {
-			return t, v, err
+			return t, v, fmt.Errorf("error parsing variable \"%s\": %w", x, err)
```

Comment:
`error parsing property %q`
It knows little about variable in the context.

Reply:
updated